sources/install/package_forensic.sh

```diff
@@ -135,6 +142,7 @@ function package_forensic() {
     install_peepdf                  # PDF analysis
     install_jadx                    # Dex to Java decompiler
     install_chainsaw                # Rapidly Search and Hunt through Windows Forensic Artefacts
+    install_magika                  # Detect file content types with deep learning
```

Comment:
This tool doesn't seem to have anything to do with adding lists. Can you remove it and make a second pull request if you want to add it for exegol ?